crates/templates/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 //! Package for working with Wasm component templates.
 
-#![deny(missing_docs)]
+#![allow(missing_docs)]
```

Comment:
This is because I'm not sure of the interface and I want to wait for feedback before writing docs... please help me remember to put it back when things firm up!